int64 utime
int64 creation_time
string channel
